change/@microsoft-teams-js-fd2fe2b1-7f23-493a-8cad-78ced187ec23.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I spent a couple minutes thinking about whether this should be a "none" change. From the consumer of teamsjs's perspective, this should result in no change to the library correct?

Reply:
(Maybe it's correct to leave this as patch since it could certainly result in a bug or change in the library. I've just been thinking lately about whether there's any utility in notifying external developers of these sorts of changes in the patch notes)

Reply:
I thought it would be a patch since there was a bug in the way it would merge capabilities before.